src/server/pfs/server/testing/server_test.go

```diff
@@ -406,6 +409,51 @@ func TestInvalidProject(t *testing.T) {
 	}
 }
 
+func TestUpdateProject_PreservesMetadata(t *testing.T) {
+	ctx := pctx.TestContext(t)
+	dbcfg := dockertestenv.NewTestDBConfig(t)
+	env := realenv.NewRealEnv(ctx, t, dbcfg.PachConfigOption)
+
```

Comment:
[nit] The PFS team generally prefers no extra newlines in function bodies, but not a hard blocker